platforms/config_settings/recycle_groups.bzl

```diff
@@ -0,0 +1,5 @@
+BUILDBUDDY_RECYCLE_GROUPS = [
+    "mysql",
+    "clickhouse",
+    "postgres",
+]
```

Comment:
Initially I was thinking that we can keep these as an internal-only config_setting. Something like `//platforms/configs:with_service_mysql`. It will provide our users with some best-practice examples down the line.

But I guess using `exec_properties` for this would be more flexible?

Reply:
We don't really know yet whether this is best practice or not - it might be better to use one service type per DB service that's running, or it might not - there are some tradeoffs that we haven't fully thought through.

So I think we should stick with setting the exec_properties (which make it feel more like a one-off thing) rather than putting it in a higher-level package that makes it look like a best practice.